datacats/error.py

```diff
@@ -25,3 +25,36 @@ def pretty_print(self):
         for line in self.message.format(*self.format_args).split('\n'):
             print "  ", line
         print colored.blue("-" * 40)
+
+
+class WebCommandError(Exception):
+
+    def __init__(self, command, container_id, logs):
+        self.command = command
+        self.container_id = container_id
+        self.logs = logs
+
+    def __str__(self):
+        return \
+            ('\nDocker container "/web" command failed\n'
+             '    Command: {0}\n'
+             '    Docker Error Log:\n'
+             '    {1}\n'
+             ).format(" ".join(self.command), self.logs, self.container_id)
+
+
+class RemoteCommandError(WebCommandError):
+    def __init__(self, base_WebCommandError):
+        self.__dict__ = base_WebCommandError.__dict__
```

Comment:
@JackMc , fortunately `__dict__` does not include methods. This is not some godless javascriptses :)

But yeah, you guys are right it is kind of ugly this way. RemoteCommandError is a subclass of WebCommandError and the two classes have the same signature. All I would like to do here is to change that class type to subclass. Not sure how to express that.

I am not convinced if passing the exception class to the function is justified. That also would look obstructified to me.